config/locales/en.yml

```diff
@@ -535,7 +535,7 @@ en:
     referred_to_clinic: Check this box if you as a case manager referred the patient to a particular clinic.
     resolved_without_fund: This is used to indicate that a patient does not require or want our services any longer.
     status_definition: Status definition
-    urgent_cases: These are patients who require a little more attention. This list is shared across all case managers working on a single line. Patients are removed from this list automatically after they are marked as having a sent pledge, or marked as resolved without assistance.
+    urgent_cases: 'These are patients who require a little more attention. This list is shared across all case managers working on a single line. Patients are removed from this list automatically after they are marked as having a sent pledge, are marked as resolved without assistance, or after %{urgent_reset} days of inactivity.'
```

Comment:
So, this isn't true. Specifically:
> Patients are removed from this list automatically after they are marked as having a sent pledge.

Nope! Nothing removes pledged patients from the list!

@colinxfleming Is this a bug in the behavior, or the documentation? I would suggest we leave the existing behavior alone (since no one has complained in the past half decade) and update this tooltip & the docs